rs/replicated_state/src/metadata_state.rs

```diff
@@ -1299,7 +1299,7 @@ impl TryFrom<pb_queues::Stream> for Stream {
             .map(|signal| {
                 Ok(RejectSignal {
                     reason: pb_queues::RejectReason::try_from(signal.reason)
-                        .map_err(ProxyDecodeError::DecodeError)?
```

Comment:
@stedfn why did we change this ?

Reply:
ok, the reason is because the return type of try_from is different from before, namely it returns [prost::UnknownEnumValue](https://docs.rs/prost/0.13.3/prost/struct.UnknownEnumValue.html)